cms/models.py

```diff
@@ -237,3 +247,47 @@ def get_context(self, request, *args, **kwargs):
         context = super().get_context(request)
 
         return context
+
+
+class AlumniPage(BootcampRunChildPage):
+    """
+    Page that holds alumni for a product
+    """
+    image = models.ForeignKey(
```

Comment:
Please rename this to `banner_image`, `poster_image` or `background_image` so that we're clear about it.